gridt/src/app/movements/movements.page.html

```diff
@@ -52,19 +52,19 @@
             <!-- Right now, this just shows every single movement -->
             <!-- Display the title on the left -->
             <ion-label text-wrap>
-              <h3>{{ movement.name }}</h3>
-              <p>{{ movement.short_description }}</p> 
+              <h3>{{ movement.name }} <ion-icon *ngIf="movement.subscribed" name="people-sharp"></ion-icon> </h3>
+              <p class="shortdescription">{{ movement.short_description }}</p> 
             </ion-label>
-          
-            <!-- Display the interval (like, weekly, monthly etc.) on the right -->
+
+             <!-- Display the interval (like, weekly, monthly etc.) on the right -->
             <ion-label slot="end"> 
               {{ movement.interval }}
-            </ion-label>
+            </ion-label>                
```

Comment:
I notice that you introduce a lot of extra spaces and the like. I notice that @marickmanrho has an option in his editor that automatically removes redundant white space. Maybe we could all enable this, because it can help a lot to keep the changes to the minimum.